docs_nnx/why.rst

```diff
@@ -114,16 +120,17 @@ upon construction and are only accesible at runtime. This makes it hard to inspe
   #   ),
   #   ...
 
-Notice that in Flax NNX there is no shape inference so both the input and output shapes must be provided
-to the Linear module. This is a tradeoff that allows for more explicit and predictable behavior.
+Notice that in the Flax NNX example above, there is no shape inference - both the input and output shapes must be provided
+to the ``Linear`` ``nnx.Module``. This is a tradeoff that allows for more explicit and predictable behavior.
 
-Running Computation
+Running computation
 ^^^^^^^^^^^^^^^^^^^
-In Flax Linen, all top-level computation must be done through the ``init`` or ``apply`` methods and the
-parameters or any other type of state is handled as a separate structure. This creates an asymmetry
-between code that runs inside ``apply`` that can run methods and other Modules directly, and code
-outside of ``apply`` that must use the ``apply`` method. In Flax NNX, there's no special context
-as parameters are held as attributes and methods can be called directly.
+
+In Flax Linen, all top-level computation must be done through the ``flax.linen.Module.init`` or ``flax.linen.Module.apply`` methods, and the
+parameters or any other type of state are handled as a separate structure. This creates an asymmetry between: 1) code that runs inside
+``apply`` that can run methods and other ``Module`` objects directly; and 2) code that runs outside of ``apply`` that must use the ``apply`` method.
+
+In Flax NNX, there's no special context because parameters are held as attributes and methods can be called directly.
```

Comment:
Good idea. Done.

 Formatting in rST may not allow for double backticks + '     ... so using "NNX Module's" with no codefont:

```
In Flax NNX, there's no special context because parameters are held as attributes and methods can be called directly. That means your NNX Module's ``__init__`` and ``__call__`` methods are not treated differently from other class methods, whereas Flax Linen Module's ``setup()`` and ``__call__`` methods are special.
```